Calculator/Calculator/ViewController.swift

```diff
@@ -0,0 +1,333 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by Leidy Luna on 15/07/23.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    
+    @IBOutlet weak var mainLabel: UILabel!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    
+    @IBAction func equal(_ sender: Any) {
+       
+       var aux = ""
+       var operationInArray: [String] = []
+        
+        for char in mainLabel.text! {
+            if char == "0" || char == "1" || char == "2" || char == "3" || char == "4" || char == "5" || char == "6" || char == "7" || char == "8" || char == "9"{
```

Comment:
use charset instead

Reply:
I used if char.isNumber || char.isPunctuation
What do you think?

Reply:
`isPunctuation` is not correct

Reply:
Yes, i changed || char == "."